baybe/surrogates/custom.py

```diff
@@ -137,3 +138,8 @@ def validate_compatibility(cls, searchspace: SearchSpace) -> None:
                 f"a one-dimensional computational representation or "
                 f"{CustomDiscreteParameter.__name__}."
             )
+
+    def __str__(self) -> str:
+        # Make a super call to get the representation of the parent class.
```

Comment:
These comments are really redundant, I have to say. Typical comment of type "Do X" and then literally do X. That's exactly the one and only purpose of `super`. Can you also remove elsewhere?
```suggestion
```